app/javascript/gobierto_data/webapp/components/commons/Queries.vue

```diff
@@ -253,11 +256,23 @@ export default {
         .catch(error => {
           const messageError = error.response.data.errors[0].sql
           this.$root.$emit('apiError', messageError)
-
           const data = []
           const keysData = []
           this.$root.$emit('sendData', keysData, data)
         })
+
+        setTimeout(() => {
+          this.showSpinner = false
+        }, 300)
```

Comment:
Why is the reason to show an spinner 300ms? Why don't you move `this.showSpinner = false` to the `then` callback? When you wanna hide the spinner actually.